File: very-easy/desafio_04.py
def num_quadrado(num):
    """Escreva  uma função que recebe um número inteiro qualquer, eleve ao quadrado cada um de seus algorismos e depois concatene o resultado retornando um único número inteiro"""
    convertString = str(num)
    number_square = ""

    for i in range(len(convertString)):
        convert_int = int(convertString[i])
        number_square += str(convert_int ** 2)

    return number_square


print(num_quadrado(25))

File: very-easy/desafio-04.js
// Escreva  uma função que recebe um número inteiro qualquer, eleve ao quadrado cada um de seus algorismos e depois concatene o resultado retornando um único número inteiro
const numQuadrado = num => {
    let convertString = num.toString();
    let numberSquare = "";

    for (let i = 0; i < convertString.length; i++) {
        numberSquare += convertString[i] ** 2;
    }
    let convertInt = Number(numberSquare);
    return convertInt;
}

console.log(numQuadrado(25));


//Solução 02

